src/SearchParameters/filterState.js

```diff
@@ -28,20 +27,20 @@ function filterState(state, filters) {
       }
 
       var numericRefinements = state.getNumericRefinements(attr);
-      if (!isEmpty(numericRefinements)) {
+      if (objectHasKeys(numericRefinements)) {
```

Comment:
I think getNumericRefinement might be returning arrays an not objects. [see test](https://github.com/algolia/algoliasearch-helper-js/blob/633c24190fb7d23d724ff6ac990116317d885dcd/test/spec/algoliasearch.helper/getNumericRefinement.js#L21)
https://github.com/algolia/algoliasearch-helper-js/blob/633c24190fb7d23d724ff6ac990116317d885dcd/test/spec/algoliasearch.helper/getNumericRefinement.js#L21

Reply:
You're looking at it with two arguments, with one argument it does actually return an object

Reply:
was already covered by existing tests :)